client/my-sites/people/delete-user/index.jsx

```diff
@@ -18,9 +18,9 @@ const Card = require( 'components/card' ),
 	FormButtonsBar = require( 'components/forms/form-buttons-bar' ),
 	AuthorSelector = require( 'blocks/author-selector' ),
 	UsersActions = require( 'lib/users/actions' ),
-	Gravatar = require( 'components/gravatar' ),
 	accept = require( 'lib/accept' ),
 	analytics = require( 'lib/analytics' );
```

Comment:
We should just follow up by porting the rest of the requires to imports in a janitorial PR. Spacing isn't a huge deal for now.